creator-node/src/routes/files.js

```diff
@@ -66,7 +66,9 @@ const streamFromFileSystem = async (
   try {
     if (checkExistence) {
       // If file cannot be found on disk, throw error
-      if (!fs.existsSync(path)) {
+      try {
+        await fs.promises.access(path)
```

Comment:
should use fsextra everywhere instead of this

Reply:
i agree. fsExtra is a bit better but mostly i like using it for standardization